integration/aggregate_documents_test.go

```diff
@@ -27,20 +28,22 @@ func TestAggregateGroupErrors(t *testing.T) {
 	t.Parallel()
 
 	for name, tc := range map[string]struct {
-		err        mongo.CommandError
-		altMessage string
-		skip       string
-		pipeline   bson.A
+		pipeline bson.A // required, aggregation pipeline stages
+
+		err        *mongo.CommandError // required
+		altMessage string              // optional, alternative error message
+		skip       string              // optional, skip test with a specified reason
 	}{
 		"StageGroupUnaryOperatorSum": {
 			pipeline: bson.A{
 				bson.D{{"$group", bson.D{{"sum", bson.D{{"$sum", bson.A{}}}}}}},
 			},
-			err: mongo.CommandError{
+			err: &mongo.CommandError{
 				Code:    40237,
 				Name:    "Location40237",
 				Message: "The $sum accumulator is a unary operator",
 			},
+			altMessage: "The $sum accumulator is a unary operator",
```

Comment:
This `altMessage` is identical to the error message.

```suggestion
```

Reply:
> Whenever altMessage is introduced, attempt to create a test case where it can be utilized. If not possible due to identical errors, include altMessage in the struct definition for potential future use.

See from https://github.com/FerretDB/FerretDB/issues/2599

When altMessage isn't used, I added it to the first test case

Reply:
I updated the struct comment too, hopefully it's more obvious. I'm still using duplicated message though, because we want every test to be potential example. Let me know if you think of a better way to demonstrate an example.

Reply:
Thanks for updating the struct comment! I think an identical `altMessage` field could be more confusing than helpful. I'd say the structure comment should be sufficient but it's up to you.